aerogear/ios-showcase-template
# AEROGEAR-3063 Gracefully fail if service configuration is not found

To be used with https://github.com/aerogear/aerogear-ios-sdk/pull/131

# Verification steps

* Clone https://github.com/aerogear/aerogear-ios-sdk/pull/131
* Run `./scripts/cloneShowcase.sh` and checkout this branch
* Run the app with the default `mobile-services.json` and ensure everything works
* Run the app with Keycloak config removed and ensure it works, but the dialog is shown on the Authenticate screen
* Run the app with Push config removed and ensure it works, but the dialog is shown on the Push Messages screen
